test/go.test.ts

```diff
@@ -482,11 +481,11 @@ It returns the number of bytes written and any write error encountered.
 			});
 		});
 
-		diffPromise.then((filePatches) => {
+		diffPromise.then((filePatches: any | FilePatch[]) => {
```

Comment:
>any [](start = 33, length = 3)

hmm why add any ? seems the resolved value to be `FilePatch[]`

Reply:
The `return reject();` lines on the promise initialization make the union necessary